com.unity.shadergraph/Editor/Generation/Processors/Generator.cs

```diff
@@ -167,11 +167,14 @@ void BuildShader()
                 Debug.LogError($"Error in Shader Graph {graphName}:{ShaderKeyword.kVariantLimitWarning}");
 
                 m_ConfiguredTextures = shaderProperties.GetConfiguredTextures();
-                m_Builder.AppendLines(ShaderGraphImporter.k_ErrorShader);
+                m_Builder.AppendLines(ShaderGraphImporter.k_ErrorShader.Replace("Hidden/GraphErrorShader2", graphName));
```

Comment:
Noticed that the error shader result here was using the wrong shader name (when variant limit was maxxed out)

Reply:
Nice catch. Should probably just update that "constant" once, but it's fine.